tests/astro_tools/test_core.py

```diff
@@ -52,21 +52,25 @@ def test_dereddener():
     testFile = os.path.join(RAILDIR, "rail", "examples_data", "testdata", "rubin_dm_dc2_example.pq")
     test_data = DS.read_file("test_data", TableHandle, testFile)
 
+    fluxToMag = LSSTFluxToMagConverter.make_stage(name='flux2mag', copy_cols=dict(ra='ra', decl='decl'))
+
     dustmap_dir = os.environ.get('RAIL_DUSTMAP_DIR')
-    is_temp_dir = False
```

Comment:
If we take this out, we might get to the `if is_temp_dir:` line without having defined `is_temp_dir`, no? (or do we get this for free with pytest?)

Reply:
yes, you are right.  we should fix this.